packages/react-components/src/organisms/__tests__/RelatedResearchCard.test.tsx

```diff
@@ -32,6 +32,26 @@ it('displays the related research card header and description', () => {
   expect(queryByText('View More Outputs')).toBeNull();
 });
 
+it('displays the related research card header and description passed by props', () => {
+  const { getByText, getByRole, queryByText } = render(
+    <RelatedResearchCard
+      {...props}
+      relatedResearch={[
+        {
+          ...createResearchOutputResponse(),
+        },
+      ]}
+      title="Related Outputs"
+      description="Find out all outputs that contributed to this one."
+    />,
+  );
+  expect(getByRole('heading', { name: 'Related Outputs' })).toBeVisible();
+  expect(
+    getByText('Find out all outputs that contributed to this one.'),
+  ).toBeVisible();
+  expect(queryByText('View More Outputs')).toBeNull();
```

Comment:
Shouldn't it be
```
expect(screen.queryByText('View More Outputs')).not.toBeInTheDocument()
```

Reply:
committed here [review: change to be null to not to be in the document](https://github.com/yldio/asap-hub/pull/3816/commits/ae9cdd871751f2c054feac65a0ca7256e9787887)